other/only-trustworthy-registries-set-root/pod-good.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: goodpod01
+  namespace: trust-registry-ns
+spec:
+  containers:
+  - name: kyverno
+    image: ghcr.io/kyverno/kyverno:latest
```

Comment:
This image does not run as root so you need an additional test here for an image which does run as root but comes from GHCR.

Reply:
```sh
crane config ghcr.io/kyverno/kyverno:latest | jq .config.User
"65532"
```

Reply:
Okay, adding `ghcr.io/nginx-proxy/nginx-proxy:latest`